.gitignore

```diff
@@ -160,11 +242,23 @@ Cargo.lock
 *.sass.map
 *.scss.map
 
-# End of https://www.gitignore.io/api/node,sass,rust,ruby
-
-# Overrides
```

Comment:
Oops these are manual overrides that need to be preserved! Sorry for missing this.